app/Http/Transformers/UsersTransformer.php

```diff
@@ -56,8 +56,8 @@ public function transformUser(User $user)
                 'permissions' => $user->decodePermissions(),
                 'activated' => ($user->activated == '1') ? true : false,
                 'ldap_import' => ($user->ldap_import == '1') ? true : false,
-                'two_factor_activated' => ($user->two_factor_active()) ? true : false,
                 'two_factor_enrolled' => ($user->two_factor_active_and_enrolled()) ? true : false,
+                'two_factor_optin' => ($user->two_factor_active()) ? true : false,
```

Comment:
I get that this is a little clearer, but IMHO it's not worth the back-compatibility break, and I would prefer we not do it.

Reply:
I really do get this argument. I have seen no evidence of anyone using this field in the API.

Reply:
(And it was wrongly-worded to start with :( )